### PR TITLE
Review messages

### PR DESCRIPTION
**Pull request by jmoreira-valory**
Review messages

### CODE REVIEW

File: run_service.sh [hunk 1]
```diff
@@ -347,7 +347,8 @@ prompt_use_staking() {
     while true; do
         echo "Use staking?"
         echo "------------"
-        read -p "Do you want to use this service in a staking program? (yes/no): " use_staking
+        echo "The current staking program for Trader service on Gnosis is 'Everest'."
```
Comment (DavidMinarsch) [inline run_service.sh:350]: This will have to be updated all the time. Don't do this. Just print the staking contract address if at all

**Review by mariapiamo — APPROVED** [review]

File: scripts/staking.py [hunk 2]
```diff
@@ -93,14 +93,14 @@
...
+                    "WARNING: Staking checkpoint call not available yet\n"
+                    "--------------------------------------------------\n"
                     f"The liveness period ({liveness_period/3600} hours) has not passed since the last checkpoint call.\n"
                     f"  - {formatted_last_ts} - Last checkpoint call.\n"
                     f"  - {formatted_next_ts} - Next checkpoint call availability.\n"
                     "\n"
-                    "If you proceed with unstaking, you will lose any rewards accrued after the last checkpoint call.\n"
-                    "Consider waiting until the liveness period has passed."
+                    "If you proceed with unstaking, your agent's work done between the last checkpoint call until now will not be accounted for rewards. "
+                    "(Note: To maximize agent work eligible for rewards, the recommended practice is to unstake shortly after a checkpoint has been called and restake immediately after.)"
```
Comment (DavidMinarsch) [inline scripts/staking.py:103]: re-stake has a different meaning. Please use "stake again"

**Review by DavidMinarsch — CHANGES_REQUESTED** [review]

File: scripts/staking.py (hunk 2 above)
Comment (jmoreira-valory) [reply scripts/staking.py]: Fixed

**Comment by dagacha** [comment]
Nice!

**Review by dagacha — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**